include/glow/Partitioner/Partitioner.h

```diff
@@ -233,6 +233,10 @@ class Partitioner {
   /// than the given physical devices.
   llvm::Error logicalDevicesValidation(NodeToFunctionMap &partitions);
 
+  /// Check if the memory usage of each partition meets the physical device
+  /// memory restriction.
+  llvm::Error memoryUsageValidation(NodeToFunctionMap &partitions);
```

Comment:
semantically this should not change partitions and any state of the class.

anything stops from this being:
llvm::Error memoryUsageValidation(```const``` NodeToFunctionMap &partitions) ```const```;

Reply:
Discussed offline. Since some of the members of class `NodeToFunctionMap` are not marked as `const`, adding `const` here would cause compile error.  Will have a separate PR  to clean up the 
`const` issue in Partitioner.